test/sql/views.sql

```diff
@@ -53,6 +54,7 @@ SELECT * FROM timescaledb_information.hypertables WHERE owner = 'super_user' ORD
 ---Add integer table --
 CREATE TABLE test_table_int(time bigint, junk int);
 SELECT create_hypertable('test_table_int', 'time', chunk_time_interval => 10);
+SELECT attach_tablespace('tablespace1', 'test_table_int');
 CREATE OR REPLACE function table_int_now() returns BIGINT LANGUAGE SQL IMMUTABLE as  'SELECT 1::BIGINT';
```

Comment:
These changes does not seem relevant for the change you're doing. Might be better to take them as a separate PR.

Reply:
It is a separate commit.

Reply:
removed this.